### PR TITLE
i18n:  new translations + cleanup

### PR DESCRIPTION
**Pull request by Akurosia**

### CODE REVIEW

File: ui/raidboss/data/04-sb/dungeon/doma_castle.ts [hunk 1]
```diff
@@ -63,10 +68,155 @@ const triggerSet: TriggerSet<Data> = {
...
+      },
+      'replaceText': {
+        '2-Tonze Magitek Missile': '大型魔导导弹',
+        'Cermet Pile': '陶瓷合金桩',
+        'Chainsaw': '链锯',
+        'Circle Of Death': '死亡回旋',
+        'Clean Cut': '激突',
+        'Delay-Action Charge': '定时炸弹',
+        'Garlean Fire': '加雷马火炎',
+        'Gunsaw': '链锯枪',
```
Comment (ShadyWhite) [inline ui/raidboss/data/04-sb/dungeon/doma_castle.ts:185]:
```suggestion
        'Gunsaw': '链锯枪',
        'Hexadrone Bits': '魔导六轮装甲浮游炮',
```

File: ui/raidboss/data/04-sb/dungeon/doma_castle.ts [hunk 2]
```diff
@@ -63,10 +68,155 @@ const triggerSet: TriggerSet<Data> = {
...
+        'Magitek Chakram': '魔导战轮',
+        'Magitek Hexadrone': '魔导六轮装甲',
+        'Magitek Rearguard': '魔导后卫',
+        'Rearguard Bit': '魔导后卫浮游炮',
+        'The Third Armory': '第三兵器库',
+        'The Training Grounds': '练兵场',
+        'The Hall Of The Scarlet Swallow': '赤燕之间',
+      },
+      'replaceText': {
+        '2-Tonze Magitek Missile': '大型魔导导弹',
```
Comment (ShadyWhite) [inline ui/raidboss/data/04-sb/dungeon/doma_castle.ts:178]:
```suggestion
        '2-Tonze Magitek Missile': '大型魔导导弹',
        'Bits Activate': '浮游炮激活',
```

File: ui/raidboss/data/04-sb/dungeon/doma_castle.ts [hunk 3]
```diff
@@ -63,10 +68,155 @@ const triggerSet: TriggerSet<Data> = {
...
+        'Cermet Pile': '陶瓷合金桩',
+        'Chainsaw': '链锯',
+        'Circle Of Death': '死亡回旋',
+        'Clean Cut': '激突',
+        'Delay-Action Charge': '定时炸弹',
+        'Garlean Fire': '加雷马火炎',
+        'Gunsaw': '链锯枪',
+        'Magitek Missiles': '魔导飞弹',
+        'Magitek Ray': '魔导激光',
```
Comment (ShadyWhite) [inline ui/raidboss/data/04-sb/dungeon/doma_castle.ts:187]:
```suggestion
        'Magitek Ray': '魔导激光',
        'Rearguard Mines': '魔导后卫炸雷',
```

File: ui/raidboss/data/04-sb/dungeon/doma_castle.ts [hunk 4]
```diff
@@ -63,10 +68,155 @@ const triggerSet: TriggerSet<Data> = {
...
+        'Garlean Fire': 'ガレアンファイア',
+        'Gunsaw': 'ガンチェーンソー',
+        'Magitek Missiles': '魔導ミサイル',
+        'Magitek Ray': '魔導レーザー',
+        'Thermobaric Charge': '気化爆弾',
+      },
+    },
+    {
+      'locale': 'cn',
+      'missingTranslations': true,
```
Comment (ShadyWhite) [inline ui/raidboss/data/04-sb/dungeon/doma_castle.ts:165]:
```suggestion
```

File: ui/raidboss/data/04-sb/dungeon/doma_castle.ts [hunk 5]
```diff
@@ -23,14 +23,19 @@ const triggerSet: TriggerSet<Data> = {
...
       },
       run: (data) => data.seenTowers = true,
       outputStrings: {
         getTower: {
           en: 'Get Tower',
-          de: 'Türme nehmen', // FIXME
+          de: 'Turm nehmen',
           fr: 'Prenez les tours', // FIXME
           ja: '塔を踏む', // FIXME
           cn: '踩塔', // FIXME
```
Comment (ShadyWhite) [inline ui/raidboss/data/04-sb/dungeon/doma_castle.ts:41]:
```suggestion
          cn: '踩塔', 
```

**Review by ShadyWhite — CHANGES_REQUESTED** [review]

File: ui/raidboss/data/04-sb/dungeon/doma_castle.ts (hunk 5 above)
Comment (MaikoTan) [reply ui/raidboss/data/04-sb/dungeon/doma_castle.ts:41]: jp is good here too. delete fixme comment please

**Pull request merged**